cluster/gce/config-default.sh

```diff
@@ -52,6 +52,9 @@ POLL_SLEEP_INTERVAL="${POLL_SLEEP_INTERVAL:-3}"
 SERVICE_CLUSTER_IP_RANGE="${SERVICE_CLUSTER_IP_RANGE:-10.0.0.0/16}"  # formerly PORTAL_NET
 ALLOCATE_NODE_CIDRS=true
 
+# Deploy a GCE L7 loadbalancer controller
+ENABLE_CLUSTER_LOADBALANCING="${KUBE_ENABLE_CLUSTER_LOADBALANCING:-glbc}"
```

Comment:
Since you;re using a value other than "true" here, you should document what cvalues are allowed.